packages/core/src/storages/utils.ts

```diff
@@ -16,11 +16,35 @@ import { Configuration } from '../configuration';
  * this method will make sure the storage is purged only once for a given execution context, so it is safe to call
  * it multiple times.
  */
-export async function purgeDefaultStorages(config = Configuration.getGlobalConfig(), client: StorageClient = config.getStorageClient(), options: {
-    onlyPurgeOnce: boolean;
-} = {
-    onlyPurgeOnce: true,
-}) {
+
+interface PurgeDefaultStorageOptions {
+    onlyPurgeOnce?: boolean;
```

Comment:
this needs to have the other two parameters as well, not just the new one. and the signature with the options as first param needs to have only one param